packages/runtime/test-runtime-utils/src/mocksForReconnection.ts

```diff
@@ -122,6 +122,21 @@ export class MockContainerRuntimeFactoryForReconnection extends MockContainerRun
 		return containerRuntime;
 	}
 
+	public getMinSeq(): number {
```

Comment:
I'm not sure this needs to be an override. The base factory will add all clients to the quorum (and never remove them), so everyone is always connected all the time. Therefore, the code below can be safely added to the base `getMinSeq` method.

The assumption here is that the quorum is accurately representing the connection state of the client but that is the responsibility of the subclass (in this case `MockContainerRuntimeFactoryForReconnection`).

Reply:
@andre4i I have moved the check as you are suggesting here.